src/base/kuc-base.ts

```diff
@@ -1,12 +1,18 @@
 import { LitElement } from "lit";
 import { v4 as uuid } from "uuid";
+import { FileItem } from "../attachment/type";
 
 type CustomEventDetail = {
   data?: string | null;
   oldValue?: string | string[];
   value?: string | string[];
   error?: string;
 };
+type AttachmentEventDetail = {
```

Comment:
the property of AttachmentEventDetail is correct ?
I don't see any file using changedFiles.
Please check it